binderhub/tests/test_build.py

```diff
@@ -94,6 +94,30 @@ async def test_build(app, needs_build, needs_launch, always_build, slug, pytestc
     assert r.url.startswith(final["url"])
 
 
+@pytest.mark.asyncio(timeout=120)
+@pytest.mark.remote
+async def test_build_fail(app, needs_build, needs_launch, always_build, pytestconfig):
+    """
+    Test build a repo that should fail immediately.
+    """
+    slug = "gist/manics/cc003c8264db20d22b5fa437770f655e"
```

Comment:
https://gist.github.com/manics/cc003c8264db20d22b5fa437770f655e
Shall I push this as a new branch `failed` in https://github.com/binderhub-ci-repos/minimal-dockerfile ?

Reply:
That sounds like a good idea